easybuild/easyconfigs/w/Wannier90/wannier90-3.1.0-gofb-2020a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'wannier90'
+version = '3.1.0'
+
+homepage = 'http://www.wannier.org/'
+description = "The Maximally-Localised Generalised Wannier Functions  calculates the counterpart of localized molecular orbitals for crystalline systems from the wavefuction."
+
+toolchain = {'name': 'gofb', 'version': '2020a'}
+toolchainopts = {'openmp': False}
+
+source_urls = ['https://github.com/wannier-developers/%(name)s/archive/']
```

Comment:
will need to use `%(name_lower)s` if it is all in lower case

Reply:
done with `%(namelower)s`